addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -134,6 +134,11 @@ QUnit.test('base rendering when chatter has no record', async function (assert)
         "Creating a new record...",
         "should have the 'Creating a new record ...' message"
     );
+    assert.strictEqual(
+        document.querySelectorAll('.o_MessageList_loadMore').length,
+        0,
```

Comment:
containsNone

Reply:
`strictEqual` is the old way of doing it that has to not be copied over new assertions

Reply:
Done !